src/views/ingresses/details/tabs/details/IngressDetailsTab.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+
+import { IoK8sApiNetworkingV1Ingress } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import { PageSection } from '@patternfly/react-core';
+import Loading from '@utils/components/Loading/Loading';
+import IngressDetailsSection from '@views/ingresses/details/tabs/details/components/IngressDetailsSection/IngressDetailsSection';
+
+import IngressRulesSection from './components/IngressRulesSection/IngressRulesSection';
+
+type IngressDetailsTabProps = {
+  obj: IoK8sApiNetworkingV1Ingress;
+};
+
+const IngressDetailsTab: FC<IngressDetailsTabProps> = ({ obj: ingress }) => {
+  if (!ingress)
+    return (
+      <PageSection>
+        <Loading />
+      </PageSection>
+    );
+
+  return (
+    <div className="co-m-pane__body">
+      <div className="row">
```

Comment:
It feels to me here we could use some nice PF component here instead...

Reply:
Done!